app/utils/prepareAnswer.js

```diff
@@ -347,10 +347,11 @@ function dailyDecStatsGraph(body, n) {
   return link;
 }
 
-function summaryGraph(body, n) {
+const QuickChart = require('quickchart-js');
```

Comment:
Please load the module always at the beginning of the file.

Reply:
Done💯